### PR TITLE
SVOM mission page

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
SVOM logo needs to be requested by Team.

I need suggestion for JSON messages Alert table, shall I make eclairs and grm as separate columns?

Screenshot 2024-04-02 at 4 39 27 PM

### CODE REVIEW

File: app/routes/missions.svom/route.mdx
Comment (lpsinger) [inline app/routes/missions.svom/route.mdx:8]:
```suggestion
# Space Variable Objects Monitor (SVOM)
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/missions.svom/route.mdx
Comment (Vidushi-GitHub) [reply app/routes/missions.svom/route.mdx:46]: Ok thanks, I will check again.

File: app/routes/missions.svom/route.mdx
Comment (Vidushi-GitHub) [reply app/routes/missions.svom/route.mdx:43]: Yes, you are right.

File: app/routes/missions.svom/route.mdx
Comment (Vidushi-GitHub) [reply app/routes/missions.svom/route.mdx:46]: Link should work after merge of https://github.com/nasa-gcn/gcn-schema/pull/126/files

Comment (lpsinger) [reply app/routes/missions.svom/route.mdx:46]: No, it will work after merging that PR _and doing a release_. So this is blocked by https://github.com/nasa-gcn/gcn-schema/pull/126.

Comment (Vidushi-GitHub) [reply app/routes/missions.svom/route.mdx:46]: Oh! I will email them. Some corrections need to be made by them.

**Comment by Vidushi-GitHub** [comment]
@lpsinger For EP, the schema and browser links are not provided, thus, I have updated SVOM one accordingly. Is this ok?

Screenshot 2024-04-11 at 4 46 19 PM

**Comment by Vidushi-GitHub** [comment]
It's pretty stale branch with lots of merge conflict. As mission team is looking into it, created a new PR #2499

**Pull request closed without merging**
